torchvision/io/image.py

```diff
@@ -1,15 +1,33 @@
 from enum import Enum
 
+import os
+import sys
 import torch
 
 from .._internally_replaced_utils import _get_extension_path
+from warnings import warn
 
 
 try:
     lib_path = _get_extension_path("image")
+    # On Windows Python-3.8+ has `os.add_dll_directory` call,
+    # which is called from _get_extension_path to configure dll search path
+    # Condition below adds a workaround for older versions by
+    # explicitly calling `LoadLibraryExW` with the following flags:
+    #  - LOAD_LIBRARY_SEARCH_DEFAULT_DIRS (0x1000)
+    #  - LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR (0x100)
+    if os.name == "nt" and sys.version_info < (3, 8):
+
+        from ctypes.windll import kernel32 as _kernel32
+        if hasattr(_kernel32, "LoadLibraryExW"):
+            _image_lib_handle = _kernel32.LoadLibraryExW(lib_path,
+                                                         None,
+                                                         0x00001100)
+        else:
+            warn("LoadLibraryExW is missing in kernel32.dll")
```

Comment:
Wouldn't it be better to move this to https://github.com/pytorch/vision/blob/57e6e30203d24d13f361da2ec98fbd8e87a2b4b3/torchvision/_internally_replaced_utils.py#L25 , so that other invocations (such as the video loading one) benefits from this improvement as well?

Reply:
Good point, but I've better add some automated PR first